woocommerce/woocommerce-ios
# Shipping Labels: Simplify creation eligibility check

Part of #6269 


### Description

Previously we had to add additional parameters to the shipping label creation eligibility check to only enable features available in different milestones (Ref: p91TBi-4pF-p2). 

In #6270  all the feature flags for Shipping Labels have been removed, which means all features of shipping labels are now enabled. This PR simplifies the eligibility check by removing all the additional parameters (since their default values are true).

### Testing instructions

Please make sure that the Create Shipping Label button is still present in the Order Details screen for orders that are eligible for creating shipping labels. Specific steps to test:
1. Make sure that your test store has installed and configured WCShip plugin.
2. Create an order on your test store.
3. On the app, navigate to Orders > select the created order.
4. On the Order Details screen, notice that the Create Shipping Label button is present. If you placed the order with cash-on-delivery payment method and set up WCPay in your store, make sure to collect payment first. After the payment is completed, the Create Shipping Label button should be present.

---
